### PR TITLE
Updating build jenkins plugins and war version

### PR DESCRIPTION
**Pull request by coryleeio**
Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A DevOps team member has approved the PR if it is code shared across multiple services and you don't own all of the services.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).
  - [ ] Think about how this change will affect Open edX operators.  Have you updated the wiki page for the next Open edX release?

### CODE REVIEW

File: playbooks/roles/jenkins_build/defaults/main.yml [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
-build_jenkins_version: jenkins_2.222.1
+build_jenkins_version: jenkins_2.222.3
 build_jenkins_jvm_args: '-Djava.awt.headless=true -Xmx16384m -DsessionTimeout=60'
 
 build_jenkins_python_versions:
   - python3.5-dev
 
 build_jenkins_configuration_scripts:
   - 1addJarsToClasspath.groovy
-  - 2checkInstalledPlugins.groovy
```
Comment (adzuci) [inline playbooks/roles/jenkins_build/defaults/main.yml:11]: Is there a way to have this default to include this check and be able to remove it for local dev easily?

**Review by adzuci — APPROVED** [review]
LGTM, I'm hopeful that removing that check won't cause us to not update plugins, but agree that it does slow down local dev.

File: playbooks/roles/jenkins_build/defaults/main.yml (hunk 1 above)
Comment (coryleeio) [reply playbooks/roles/jenkins_build/defaults/main.yml:11]: I dont think it is a meaningful check after updating the ansible to purge that directory on run, I'm fairly sure that is why it was added in the first place

Comment (coryleeio) [reply playbooks/roles/jenkins_build/defaults/main.yml:11]: my change should make the plugins directory idempotent

**Pull request merged**